frontend/src/components/Card/CardWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+// *EXTERNALS*
+import React, { memo } from 'react';
+import useSWR from 'swr';
+
+// *INTERNALS*
+import { NFT } from '../../types';
+import { API_ADDRESS } from '../../constants';
+import NFTCard from './NFTCard';
+import SmallNFTCard from './SmallNFTCard';
+
+interface Props {
+  NFT: NFT;
+  isLoot?: boolean;
+  isSmall?: boolean;
+  disabledFlip?: boolean;
+}
+
+const fetcher = (params: any) => fetch(params).then(async (res) => URL.createObjectURL(await res.blob()));
+
+const CardWrapper = ({ NFT, isLoot, isSmall, disabledFlip }: Props) => {
+  const { name, tokenId } = NFT;
+  const actualTokenId = tokenId || +name.replace(/^\D+/g, ''); // Trick to bypass the issue of tokenId not being set in the NFT object
```

Comment:
when does this happen?

Reply:
Well the NFT object has no tokenId object, typically the caller of CardWrapper passes an NFT object that was manually extended with the tokenId